apps/web/src/server/app/validation/validation.pipes.js

```diff
@@ -75,3 +70,46 @@ export const validateOutcomeIncompletePipe = () => [
 		.isLength({ min: 1, max: 500 })
 		.withMessage('Word count exceeded')
 ];
+
+/**
+ * Validate the outcome incomplete form to ensure it has at least 1 answer.
+ *
+ * @returns {void}
+ */
+export const validateOutcomeInvalidReason = () => [
+	body('invalidReasons')
+		.notEmpty()
+		.withMessage('Please select a reason why the appeal is invalid')
+		.bail()
+		.isIn([
+			'outOfTime',
+			'notApplicable',
+			'noRightOfAppeal',
+			'lpaInvalid',
+			'other'
+		])
+		.withMessage('Please enter a reason why the appeal is invalid'),
+	body('otherReason')
+		.if(body('invalidReasons').custom(makeValidator_StringMatchesOrArrayContainsMatch('other')))
```

Comment:
This will need replacing with `.if(body('invalidReasons').toArray().custom((value) => value.includes('other')))`